src/Template/Requests/view.ctp

```diff
@@ -22,7 +24,8 @@ use Cake\Routing\Router;
     </li>
     <?php endforeach; ?>
     <li id="panel-button">
-        <?= $this->Html->image('DebugKit.cake.icon.png', ['alt' => 'Debug Kit']) ?>
+        <?= $this->Html->image('DebugKit.cake.icon.png',
+			['alt' => 'Debug Kit', 'title' => 'CakePHP ' . Configure::version() . ' Debug Kit']) ?>
```

Comment:
The CakePHP version is already show in the Environment panel, why duplicate that here?